src/Checksum_Plugin_Command.php

```diff
@@ -269,7 +269,7 @@ private function get_plugin_files( $path ) {
 	 */
 	private function check_file_checksum( $path, $checksums ) {
 		if ( $this->supports_sha256()
-		     && array_key_exists( 'sha256', $checksums ) ) {
+			&& array_key_exists( 'sha256', $checksums ) ) {
```

Comment:
Just a suggestion for readability.
```suggestion
			&& array_key_exists( 'sha256', $checksums ) 
		) {
```